frontend/src/core/codemirror/cells/extensions.ts

```diff
@@ -236,6 +240,21 @@ export function cellMovementBundle(
         return true;
       },
     },
+    {
+      key: HOTKEYS.getHotkey("cell.splitCell").key,
+      preventDefault: true,
+      stopPropagation: true,
+      run: (ev) => {
+        const cursorPos = ev.state.selection.main.head;
+        splitCell({ cellId, cursorPos }),
```

Comment:
i think you want a `;` at the end of the line: `splitCell({ cellId, cursorPos });`